bve-native/include/bve_prefix.h

```diff
@@ -9,3 +9,22 @@
 #else
 	#define BVE_NO_DISCARD
 #endif
+
+typedef struct {
+    float x;
+    float y;
+} BVE_Vec2;
+
+typedef struct {
+    float x;
+    float y;
+    float z;
+    float _z;
```

Comment:
I thought you renamed this to `_padding`?

Reply:
Visual studio doesn't autosave. I'm used to clion which does.